### ISSUE TITLE
Triple brackets inside 'each' loop

### ISSUE THREAD

**Issue by belousandrey** [doy/text-handlebars#6]
Found following problem: if I use triple brackets inside {{#each}}, I get the same result as with double brackets.
```perl
my $vars = { outer => '<em>example</em>', elements => [ { inner => '<em>text</em>' }, { inner => '<h1>text</h1>' } ] };
my $template = <<EOL;
{{{outer}}}
{{#each elements}}
{{{inner}}}
{{/each}}
EOL
```
Is it a bug? Or I use it wrong?

**Comment by jdigory** [comment]
This works in http://tryhandlebarsjs.com/ with template of:
`{{#each posts}}
   <p>{{{content}}}</p>
{{/each}}`
and context of:
`{ 
  posts: [
    { content: "<i>foo</i>" },
    { content: "bar" }
  ]
}`
so it would seem to be a bug in Text::Handlebars

**Comment by doy** [comment]
Fixed in 0.05, thanks!
